docs/index.rst

```diff
@@ -4,7 +4,7 @@ python-gvm: Python API for the Greenbone Vulnerability Manager
 **python-gvm** is the official python library to control the Greenbone
 Vulnerability Manager (GVM) remotely.
 
-.. note:: **python-gvm** requires at least Python 3.5. Python 2 is not supported.
+.. note:: **python-gvm** requires at least Python 3.7. Python 2 is not supported.
```

Comment:
This should be put into an extra pr because i guess we need to release a 20.8.1 that still needs to support python 3.5.
```suggestion
.. note:: **python-gvm** requires at least Python 3.5. Python 2 is not supported.
```

Reply:
Sure ...